src/main/java/frc/robot/commands/elevator/MoveToHeight.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.commands.elevator;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Elevator;
+
+public class MoveToHeight extends CommandBase {
+  Elevator m_elevator; 
+  double m_targetHeight; 
+  /**
+   * This command allows the elvator to move to the heights 
+   * of the bottom, middle, and top nodes, or any height that we choose. 
+   * 
+   * @param elevator
+   * @param targetHeight
+   */
+  public MoveToHeight(Elevator elevator, double targetHeight) {
+    m_elevator = elevator; 
+    m_targetHeight = targetHeight; 
+    addRequirements(m_elevator);
+  }
+
+  @Override
+  public void initialize(){
+    //set the elevator PID speed to be clamped within the correct range to get it to move faster
+    //This should be done already in ResetEncoderAtBottom() but just in case something happens it's here too. 
+    m_elevator.setSetpoint(m_targetHeight); 
+    m_elevator.setEnabled(true);
+  }
+
+  @Override
+  public void end(boolean interrupted) {
+    System.out.println("MoveTOHeight Command has FINISHEDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD"); 
+    
+   
+  }
```

Comment:
remove random spaces and the print at end. The print will clutter the rio log when if you run this command multiple times wich prosumably you will to a lot durring a match. Additionaly generaly the command shouldn't print just to signal it is finished. If you aren't going to use this method for anything I would suggest that you remove it.